applications/xcm-tools.md

```diff
@@ -0,0 +1,171 @@
+# XCM Tools
+
+* **Team Name:** [gmajor](https://github.com/gmajor-encrypt)
+* **Payment Address:** 0xC3094f0ddce699a1Ad9Ef2621DF68Cd297a4c44F(USDC)
+* **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+RFPs [https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Open/xcm-tool.md](https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Open/xcm-tool.md)
+
+### Overview
+
+XCM (Cross-Consensus Message) is a cross-chain message between parachain and relaychain because the xcm format (
+Instruction, MultiAssets, and MultiLocation) are too complex, and no good tool exists. It has yet to be widely used.
+
+XCM Tools is a tool and an SDK. This library is written in golang. It provides the following functions: sending xcm
+messages, parsing xcm message instructions and getting the execution result of the execution after sending xcm.
+
+### Project Details
+
+We only have two libs to choose from, and some functions still need to be included.
+
+[Go-Substrate-RPC-Client](https://github.com/centrifuge/go-substrate-rpc-client)
+Centrifuge's open source and supports transaction and chain status queries. Scale Codec is
+a [static](https://pkg.go.dev/github.com/centrifuge/go-substrate-rpc-client#hdr-Types) and strongly typed library.
+All types of analysis require user definition.
+
+[Substrate-API-rpc](https://github.com/itering/substrate-api-rpc), [scale.go](https://github.com/itering/scale.go)
+Subscan open source, support chain status query, Scale Codec supports dynamic types, good serialization, and back-order
+functions, history version types support. But lack the function of transaction sending.
+
+After comparison, I chose [Substrate-API-rpc](https://github.com/itering/substrate-api-RPC) as dependence, but because
+of the lack of the function of sending transactions, I also need to change dependence.
```

Comment:
```suggestion
```suggestion
After comparing it with similar libraries, I decided to use [Substrate-API-rpc](https://github.com/itering/substrate-api-RPC). However, I discovered that it does not have the capability to send transactions. As a result, I'm planning to implement this functionality into the library.
```
```